docs/background/community.rst

```diff
@@ -5,12 +5,8 @@ Briefcase is part of the `BeeWare suite`_. You can talk to the community
 through:
 
 * `@pybeeware on Twitter`_
-
-* The `BeeWare Users Mailing list`_, for questions about how to use the
-* BeeWare suite.
-
-* The `BeeWare Developers Mailing list`_, for discussing the development of
-* new features in the BeeWare suite, and ideas for new tools for the suite.
+* The `BeeWare Users Mailing list`_, for questions about how to use the BeeWare suite.
+* The `BeeWare Developers Mailing list`_, for discussing the development of new features in the BeeWare suite, and ideas for new tools for the suite.
```

Comment:
Both of these mailing lists say they're "closed". Should they just be removed?

Reply:
Doh, I see this was done in #230. Should have read the rest of my notifications first...